examples/blog/config.js

```diff
@@ -14,6 +14,14 @@
             return {configuration: null};
         },
         componentDidMount: function () {
+            var restful = this.refs.admin.state.restful;
```

Comment:
As there is no service or DI in ReactJS, I don't know how to do better.

We can do a getter in the `ReactAdmin` component but it will not be more elegant:
`var restful = this.refs.admin.getRestful();`